src/index.js

```diff
@@ -14,7 +14,7 @@ const store = createStore(rootReducer, devTools);
 ReactDOM.render(
   <Provider store={store}>
     <BrowserRouter>
-      <App />
+      <Root />
```

Comment:
App is now a child of Root. After our discussion, let's work off the conclusion that our path changing was not triggering a re-render due to neither props nor state being updated (per React standards). If Root has App as a child, it should be able to pass and maybe alter props (still semi-unsure how), but a re-render is now triggered likely due to interacting with the props. Pasting in the relevant docs link below for future reference:
https://redux.js.org/advanced/usage-with-react-router#usage-with-react-router
Let's continue digging deeper into why this works/if it's the optimal solution tomorrow.